tests/src/app/testqgsmaptoolrotatefeature.cpp

```diff
@@ -137,8 +139,8 @@ void TestQgsMapToolRotateFeature::testRotateFeature()
   utils.mouseMove( 2, 1 );
   utils.mouseClick( 2, 1, Qt::LeftButton, Qt::KeyboardModifiers(), true );
 
-  QCOMPARE( mLayerBase->getFeature( 1 ).geometry().asWkt( 2 ), QStringLiteral( "Polygon ((0.72 -0.17, 0.28 1.17, 1.17 0.72, 0.72 -0.17))" ) );
-  QCOMPARE( mLayerBase->getFeature( 2 ).geometry().asWkt( 2 ), QStringLiteral( "Polygon ((1.1 0, 1.1 5, 2.1 5, 2.1 0))" ) );
+  QCOMPARE( mLayerBase->getFeature( 1 ).geometry().asWkt( 2 ), QStringLiteral( "Polygon ((-0.17 0.28, 0.28 1.17, 1.17 0.72, 0.72 -0.17, -0.17 0.28))" ) );
```

Comment:
Why have these values changed?

Reply:
The initial polygons wkt1 and wkt2 were invalid, they were open rings missing their last vertex. As a result, when those squares were rotated, a triangle was produced. In this commit I added the missing vertex to the polygons so the rotated geometries became squares too. I generally don't like modifying tests, but this one made no sense!